src/services/Provider.ts

```diff
@@ -194,7 +196,7 @@ export class Provider {
    * @return {Promise<FileInfo[]>} urlDetails
    */
   public async checkFileUrl(
-    url: string,
+    file: UrlFile | Arweave,
```

Comment:
why only adding `Arweave`? There are many more types in `@types/Asset` so not clear why only one of those is added.

Reply:
I only added Arweave in the interest of keeping this PR as small and focused as possible.